Resources/doc/index.rst

```diff
@@ -57,6 +57,13 @@ command from the directory where your ``composer.json`` file is located:
 Now, Composer will automatically download all required files, and install them
 for you.
 
+.. note::
+
+    There are two drivers of Mongo Database for Php: mongo (older, not maintained,
```

Comment:
To not maintain two PHP 7 related sections could you please rephrase this to something like:

Currently ODM works with legacy MongoDB driver (i.e. `ext-mongo`) which is not compatible with PHP 7. If you have PHP 7 and/or are using new driver (i.e. `ext-mongodb`) please refer to ["Using PHP 7" section](http://docs.doctrine-project.org/projects/doctrine-mongodb-odm/en/latest/reference/introduction.html#using-php-7) in ODM's documentation.